frontend/src/store.js

```diff
@@ -30,7 +30,7 @@ const init = {
   questionNumber: 0,
   closedQuestions: closedQuestions,
   openExportModal: false,
-  currentQuestionInfo: baseQuestionInfo,
+  currentQuestionInfo: JSON.parse(JSON.stringify(baseQuestionInfo)),
```

Comment:
Doesn't doing `JSON.parse(JSON.stringify(stuff))` just give you what it was originally?

Reply:
yes. 

previously the `baseQuestionInfo` and `currentQuestionInfo` were both referencing the same object, which causes the changes done to the answer options array in `currentQuestionInfo` be reflected in `baseQuestionInfo`. 

this way it's creating a copy of the `baseQuestionInfo` object instead which stops them from being linked.

Reply:
actually now that you commented I realized I also need to do this in reset

Reply:
Yeah idk whats doing on here

Reply:
@weining-li then using `Object.assign({}, whatever)`

Reply:
Ok cool

Reply:
> @weining-li then using `Object.assign({}, whatever)`

I personally would have used `Object.assign`. But both do the same thing so 🤷

Reply:
LJ said we need to deep copy but `Object.assign` is shallow copy.

Reply:
U right. I thought `Object.assign` did deep copy but it doesn't